src/screens/Lpn/Create.tsx

```diff
@@ -177,7 +177,7 @@ class CreateLpn extends React.Component<Props, State> {
 }
 
 const mapStateToProps = (state: RootState) => ({
-  currentLocation: state.mainReducer.currentLocation,
+  currentLocation: state.locationsReducer.SelectedLocation,
```

Comment:
Why there are two reducers that are supposed to keep the same thing? Remove one of these and keep only one `currentLocation` reducer. This will help avoid these types of issues in the future.